BUFRCREX_TableB_en_11.csv

```diff
@@ -18,6 +18,8 @@ ClassNo,ClassName_en,FXY,ElementName_en,BUFR_Unit,BUFR_Scale,BUFR_ReferenceValue
 11,Wind and turbulence,011021,Relative vorticity,/s,9,-65536,17,/s,9,6,,,Operational
 11,Wind and turbulence,011022,Divergence,/s,9,-65536,17,/s,9,6,,,Operational
 11,Wind and turbulence,011023,Velocity potential,m2/s,-2,-65536,17,m2/s,-2,6,,,Operational
+11,Wind and turbulence,011024,Average u-component after the vortex removal,m/s,1,-4096,13,m/s,0,0,,,Operational
+11,Wind and turbulence,011024,Average v-component after the vortex removal,m/s,1,-4096,13,m/s,0,0,,,Operational
```

Comment:
```suggestion
11,Wind and turbulence,011025,Average v-component after the vortex removal,m/s,1,-4096,13,m/s,0,0,,,Operational
```

Reply:
@marijanacrepulja should this be 011025?

Reply:
@amilan17 yes, well spotted. Thanks!